src/SDKs/ServiceBus/ServiceBus.Tests/ServiceBus.Tests.csproj

```diff
@@ -10,6 +10,12 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Including ItemGroup for files is not necessary.
Please remove this ItemGroup as the new project system picks up all files

Reply:
Removed.